src/datalink/bpf.rs

```diff
@@ -19,17 +19,17 @@ use std::sync::Arc;
 use bindings::bpf;
 use packet::Packet;
 use packet::ethernet::{EthernetPacket, MutableEthernetPacket};
-use datalink::DataLinkChannelType;
+use datalink::{DataLinkChannelType,DataLinkSender,DataLinkReceiver,DataLinkChannelIterator};
 use datalink::DataLinkChannelType::{Layer2, Layer3};
 use internal;
 use util::NetworkInterface;
 
 // NOTE buffer must be word aligned.
-pub fn datalink_channel(network_interface: &NetworkInterface,
+pub fn datalink_channel_bpf(network_interface: &NetworkInterface,
```

Comment:
The trick here is:

```rust
use pnet::datalink::netmap;

netmap::datalink_channel(...)
```

I would eventually also like to drop the `datalink_` prefix, but that needs greater design thought (I want to get rid of most of the parameters and just pass in an interface and some configuration, I haven't figured out the right way to do that though - may as well do both breaking changes at once).